src/search/landmarks/landmark_cost_assignment.cc

```diff
@@ -17,12 +17,12 @@
 using namespace std;
 
 namespace landmarks {
-LandmarkCostAssignment::LandmarkCostAssignment(
+LandmarkCostPartitioningAlgorithm::LandmarkCostPartitioningAlgorithm(
```

Comment:
The other classes don't have the `Landmark` prefix, so I'd drop it here as well.